recipes/freetype/meson/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.build import can_run
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if can_run(self):
+            bin_path = os.path.join(self.cpp.build.bindirs[0], "test_package")
+            font_path = os.path.join(self.source_folder, "OpenSans-Bold.ttf")
```

Comment:
OpenSans License should be mentioned somewhere (both in `all/` and `meson/`)

Reply:
It might be acceptable to just make sure the license is embedded in the font: https://openfontlicense.org/how-to-use-ofl-fonts/